### ISSUE TITLE
Can't move files to usb drive

### ISSUE THREAD

**Issue by cm-mc** [zhanghai/MaterialFiles#1254]
Android 10, one UI 2.5
v.1.7.2
Root auto
Can access source and destination (through paragon app), but can't move files from sd card to usb drive (there's disk activity, but nothing happens after selecting destination).

**Comment by Feuerswut** [comment]
Does the app have access to the USB device, it may have to be granted after plugging it in. (each time)

**Comment by cm-mc** [comment]
Yes, and i can read the device slright; the permission is granted to `content://com.paragon_software.documentproviderserver.documents/tree/root`; this works on every other app (fx, x-plorer, mixplorer, even android file transfer app). Anyway, just tried again with just one file (instead of dozens), and without any settings change, and it worked; maybe there's some file number/total file size limit?

**Comment by Feuerswut** [comment]
> Yes, and i can read the device alright; the permission is granted to `content://com.paragon_software.documentproviderserver.documents/tree/root`; this works on every other app (fx, x-plorer, mixplorer, even android file transfer app). Anyway, just tried again with just one file (instead of dozens), and without any settings change, and it worked; maybe there's some file number/total file size limit?

This isn't paragon explorer.
The app id is **com.zanghai.materialfiles**

This app needs to be granted access via the android documentsui (default file picker) to the usb stick.

make a screenshot of the error you encounter please and provide it here.

**Comment by cm-mc** [comment]
Permission is granted BY documentsui, TO Paragon. Paragon provides read and WRITE access to NTFS, etc., and it absolutely works.
My phone is a Samsung and the drive is a 8TB NTFS. I HAVE read and WRITE access to the USB drive, through either Material Files, FX, X-plore, Mixplorer, Total Commander, Documentsui, etc.
There was NO error code; it simply just didn't move any of the dozens files I selected to.
![Screenshot_20240624-092413_Material Files](https://github.com/zhanghai/MaterialFiles/assets/11512937/bd94deb6-8665-49aa-849a-c36e7c105c56)
